### DISCUSSION TITLE
Why can't I get the unread message?

### DISCUSSION THREAD

**Discussion by SimFG** [community/community#112396] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/21985684/9069d744-622d-4a16-a076-f7e9e54dd1d4)
How can i make it disappear? I force to clean all message, but it's still here

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @SimFG , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by SimFG** [comment]
@queenofcorgis thanks a lot for your changes
